tests/test_printer.py

```diff
@@ -59,6 +59,24 @@ def test_simple_forgotten_op():
     assert_print_op(add, expected, None)
 
 
+def test_simple_forgotten_op():
+    """
+    Test that the parsing of an undefined operand does raises an exception if
```

Comment:
Should this be: `does not raise an exception`?

Also, in my current terminal it does print this error without the proposed change, but it does not seem to actually raise an exception?

Reply:
Oops, I changed the documentation, thanks!
The reason why you don't see the error is because you use `print_op`, which does not print a newline at the printer level.
Because of that, it does not print the error at the next line.
This is utterly broken, and I want to fix that at some point! Here, what changes is that if you print multiple operations, they won't show the error at all.

Reply:
> The reason why you don't see the error
I do see the error no worries! But now the docs make more sense!
Thank you!